sys-process/cronbase/cronbase-0.3.7-r10.ebuild

```diff
@@ -25,3 +25,12 @@ src_install() {
 
 	keepdir /var/spool/cron/lastrun
 }
+
+pkg_postinst() {
+	elog "To add a user to the cron group so it can create cron jobs, run:"
```

Comment:
I am not part of Gentoo's base system team, but I would consider showing this message only on the first installation (and potentially using `readme.gentoo-r1.eclass`).

Reply:
Changed it in latest commit. Thanks for the feedback